api/v1beta2/provider_types.go

```diff
@@ -24,29 +24,30 @@ import (
 )
 
 const (
-	ProviderKind           string = "Provider"
-	GenericProvider        string = "generic"
-	GenericHMACProvider    string = "generic-hmac"
-	SlackProvider          string = "slack"
-	GrafanaProvider        string = "grafana"
-	DiscordProvider        string = "discord"
-	MSTeamsProvider        string = "msteams"
-	RocketProvider         string = "rocket"
-	GitHubDispatchProvider string = "githubdispatch"
-	GitHubProvider         string = "github"
-	GitLabProvider         string = "gitlab"
-	GiteaProvider          string = "gitea"
-	BitbucketProvider      string = "bitbucket"
-	AzureDevOpsProvider    string = "azuredevops"
-	GoogleChatProvider     string = "googlechat"
-	WebexProvider          string = "webex"
-	SentryProvider         string = "sentry"
-	AzureEventHubProvider  string = "azureeventhub"
-	TelegramProvider       string = "telegram"
-	LarkProvider           string = "lark"
-	Matrix                 string = "matrix"
-	OpsgenieProvider       string = "opsgenie"
-	AlertManagerProvider   string = "alertmanager"
+	ProviderKind              string = "Provider"
+	GenericProvider           string = "generic"
+	GenericHMACProvider       string = "generic-hmac"
+	SlackProvider             string = "slack"
+	GrafanaProvider           string = "grafana"
+	DiscordProvider           string = "discord"
+	MSTeamsProvider           string = "msteams"
+	RocketProvider            string = "rocket"
+	GitHubDispatchProvider    string = "githubdispatch"
+	GitHubProvider            string = "github"
+	GitLabProvider            string = "gitlab"
+	GiteaProvider             string = "gitea"
+	BitbucketProvider         string = "bitbucket"
+	AzureDevOpsProvider       string = "azuredevops"
+	GoogleChatProvider        string = "googlechat"
+	GooglePubSubTopicProvider string = "googlepubsubtopic"
```

Comment:
Maybe we can leave it at `googlepubsub`
```suggestion
	GooglePubSubProvider string = "googlepubsub"
```

Reply:
Fixed, thanks!